pkg/operator/collection.go

```diff
@@ -320,6 +320,31 @@ func gzipData(data []byte) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+func setConfigMapData(cm *corev1.ConfigMap, c monitoringv1.CompressionType, fn string, data string) error {
```

Comment:
suggestion (non-blocking): `fn` threw me off for a minute as I was reading `function`. Maybe `filename` or `path`?